python/pyspark/sql/group.py

```diff
@@ -194,6 +194,37 @@ def pivot(self, pivot_col, values=None):
             jgd = self._jgd.pivot(pivot_col, values)
         return GroupedData(jgd, self.sql_ctx)
 
+    def apply(self, udf_obj):
+        """
+        Maps each group of the current [[DataFrame]] using a pandas udf and returns the result
```

Comment:
little nit: `[[DataFrame]]` to  ``:class:`DataFrame` ``

Reply:
Fixed